src/extensions/timeseries/TimeWidgetNew/TimeWidgetNew.tsx

```diff
@@ -197,13 +201,47 @@ const plotMargins = {
     bottom: 100
 }
 
+const useTimeState = (args: {
```

Comment:
This custom hook needs to be simplified. I was trying to achieve both controlled and uncontrolled state for the recording selection. Not very elegant.

Reply:
Sorry, I'm not sure I follow what you mean by 'controlled and uncontrolled state' in this context.